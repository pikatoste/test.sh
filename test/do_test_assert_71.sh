#!/bin/bash
test_01() {
  assert_true false "test_01"
}

teardown_test() {
  assert_true false "teardown_test"
}

teardown_test_suite() {
  assert_true false "teardown_test_suite"
}

source "$(dirname "$(readlink -f "$0")")"/../test.sh

result_of 'assert_true false "first"'
result_of 'assert_true false "second"'

run_tests
