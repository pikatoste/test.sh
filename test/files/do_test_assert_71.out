[test.sh] assert exception: Assertion failed: first, expected success but got failure in: 'false'
[test.sh] Caused by:
[test.sh] implicit exception: Error in _eval(test.sh:): 'false' exited with status 1
[test.sh] assert exception: Assertion failed: second, expected success but got failure in: 'false'
[test.sh] Caused by:
[test.sh] implicit exception: Error in _eval(test.sh:): 'false' exited with status 1
[test.sh] Start test: test_01
[test.sh] assert exception: Assertion failed: test_01, expected success but got failure in: 'false'
[test.sh] Caused by:
[test.sh] implicit exception: Error in _eval(test.sh:): 'false' exited with status 1
[test.sh] FAILED: test_01
[test.sh] assert exception: Assertion failed: teardown_test, expected success but got failure in: 'false'
[test.sh] Caused by:
[test.sh] implicit exception: Error in _eval(test.sh:): 'false' exited with status 1
[test.sh] assert exception: Assertion failed: teardown_test_suite, expected success but got failure in: 'false'
[test.sh] Caused by:
[test.sh] implicit exception: Error in _eval(test.sh:): 'false' exited with status 1
[test.sh] 1 test(s) failed
